src/tokens/utilities.ts

```diff
@@ -1,5 +1,19 @@
+import type {TokenGroup} from './tokens';
+
 const BASE_FONT_SIZE = 16;
 
-export function rem(px: string) {
-  return `${parseInt(px, 10) / BASE_FONT_SIZE}rem`;
+function rem(value: string) {
+  return value.replace(
+    // https://regex101.com/r/RBL7EE/1
```

Comment:
These links are temporary right? Not sure it's worth leaving as a comment. Easy enough just to copy and paste and add a few examples

Reply:
![image](https://user-images.githubusercontent.com/32409546/145494676-2913a42c-4fac-4403-be7f-c1b232fc5f7c.png)
In addition to the above screenshot, I verified in their GitHub API and FAQ wikis these are permalinks. I actually like having this reference as we can make sure the existing test cases still pass during future updates. That said, I'm also fine with removing the link as it's now checked into source control if we ever need it.